public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -12,6 +14,23 @@ import { onRemovePanel, toggleVizPanelLegend } from './PanelMenuBehavior';
 
 export function setupKeyboardShortcuts(scene: DashboardScene) {
   const keybindings = new KeybindingSet();
+  let vizPanelKey: string | null = null;
+
+  const panelAttentionSubscription = appEvents.getStream(SetPanelAttentionEvent).subscribe((event) => {
+    if (typeof event.payload.panelId === 'string') {
+      vizPanelKey = event.payload.panelId;
+    }
+  });
+
+  function withFocusedPanel(scene: DashboardScene, fn: (vizPanel: VizPanel) => void) {
+    return () => {
+      const vizPanel = sceneGraph.findObject(scene, (o) => o.state.key === vizPanelKey);
+      if (vizPanel && vizPanel instanceof VizPanel) {
+        fn(vizPanel);
+        return;
+      }
+    };
+  }
```

Comment:
I consider removing this, as the imports in the PRs are not working correctly until Scenes is pointing to the correct version of grafana-data and grafana-ui.